spring-cloud-gcp-data-spanner/src/main/java/org/springframework/cloud/gcp/data/spanner/core/convert/ConversionUtils.java

```diff
@@ -54,4 +60,46 @@ public static boolean isIterableNonByteArrayType(Class propType) {
 		source.forEach((item) -> result.add(converter.convert(item, targetType)));
 		return result;
 	}
+
+	public static <T> T wrapSimpleLazyProxy(Supplier<T> supplierFunc, Class<T> type) {
+		return (T) Proxy.newProxyInstance(type.getClassLoader(), new Class[] { type },
+				new SimpleLazyDynamicInvocationHandler<>(supplierFunc));
+	}
+
+	public static boolean isUnevaluatedLazyProxy(Object object) {
+		if (!Proxy.isProxyClass(object.getClass()) ||
```

Comment:
maybe simplify this method to 
```
public static boolean isUnevaluatedLazyProxy(Object object) {
if (Proxy.isProxyClass(object.getClass()) && (Proxy.getInvocationHandler(object) instanceof SimpleLazyDynamicInvocationHandler)) {
SimpleLazyDynamicInvocationHandler handler = (SimpleLazyDynamicInvocationHandler) Proxy
				.getInvocationHandler(object);
		return !handler.isEvaluated();
}
return false;
}
```
to make it more readable?

Reply:
done.